test_data/asm/book/assert_write_register.asm

```diff
@@ -0,0 +1,22 @@
+machine Machine {
+
+    degree 8;
+
+    // ANCHOR: component
+reg pc[@pc];
```

Comment:
Indentation is a bit off, not sure if that matters.

Reply:
So this is because sections of this file are included in code blocks in the book. They looked better without indentation. There's a dilemma though, because sometimes we also want to show the whole file, and only one of the two would look nice. I defaulted to keeping the indentation in these cases, see the function page

Reply:
sg